quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow-integration-test/src/test/java/org/kie/kogito/quarkus/workflows/ForEachRestIT.java

```diff
@@ -19,23 +19,34 @@
 
 import org.junit.jupiter.api.Test;
 
+import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.junit.QuarkusIntegrationTest;
 import io.restassured.http.ContentType;
 
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.nullValue;
 
+@QuarkusTestResource(DivisionMockService.class)
 @QuarkusIntegrationTest
 class ForEachRestIT {
 
+    @Test
+    void testForEachRpc() {
+        testForEachWorkItem("/forEachCustomType");
+    }
+
     @Test
     void testForEachRest() {
+        testForEachWorkItem("/forEachRest");
+    }
+
+    private void testForEachWorkItem(String id) {
```

Comment:
You can simplify it by using parameters:

```suggestion
    @ParameterizedTest
    @ValueSource(strings = { "/forEachCustomType", "/forEachRest" })
    void testForEachWorkItem(String id) {
```